bin/conpot

```diff
@@ -221,10 +221,9 @@ def main():
                   ' running Conpot')
             sys.exit(3)
         try:
-            if not os.path.isfile(os.path.join(package_directory, args.config)):
+            if not os.path.join(args.config):
```

Comment:
I don't think this would check whether the file exists or not. It would return `True` as long as a string is returned by `os.path.join`

Reply:
Thanks for the correction, my idea was to write `if not os.path.isfile(os.path.join(args.config)):` but I misspelled it. I´ve closed the PR as we spoke in the issue.